lib/http/chainable.rb

```diff
@@ -6,42 +6,48 @@
 
 module HTTP
   module Chainable
-    # Request a get sans response body
+    # Request a get sans response body.
+    # See Client#request.
     # @param uri
     # @option options [Hash]
     def head(uri, options = {}) # rubocop:disable Style/OptionHash
       request :head, uri, options
     end
 
-    # Get a resource
+    # Get a resource.
+    # See Client#request.
```

Comment:
Please use `@see` tag

```suggestion
    # @see Client#request
```